tests/ICU4N.Tests/Dev/Test/Lang/UCharacterTest.cs

```diff
@@ -3032,7 +3032,13 @@ public void TestBlockData()
 
                 try
                 {
-                    if ((int)ubc.GetTypeInfo().GetField(name + "_ID").GetValue(blk) != b)
+                    int fieldValue =
```

Comment:
Please change this block to the following:

```c#
                    if ((int)ubc.GetField(name + "_ID").GetValue(blk) != b)
```

We don't need a conditional block here, as this line compiles fine on .NET Standard 1.x.

Reply:
Okay, i will fix this in the next commit.